sources/@roots/bud/bin/bud.js

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env node
+// @ts-check
 
-const {run, flush, Errors} = require('@oclif/core')
+const [node, app, ...args] = process.argv;
 
-run().then(flush).catch(Errors.handle)
+(async () => {
+  const {Cli} = await import('clipanion');
+  const {BuildCommand, CleanCommand, DevCommand, DoctorCommand} = await import('../lib/cjs/cli/index.js');
+  
+  const cli = new Cli({
+    binaryLabel: `bud`,
+    binaryName: `bud`,
+    binaryVersion: `1.0.0`,
```

Comment:
should this be bud's version number? or what is this?

Reply:
yes, it should. i'm just gonna read `package.json`

Reply:
fixed in [42c1c33](https://github.com/roots/bud/pull/1153/commits/42c1c33c0977a83af4dcb41b71ecb5a961d4b2c0)

Reply:
actually i didn't `git add` this change so it's [da664eb](https://github.com/roots/bud/pull/1153/commits/da664eba0fd3de21419e60aa6067fad6b450b035)